wiki/Contributing.md

```diff
@@ -0,0 +1,86 @@
+# Contributing
+
+Thank you for considering contributing to our project! We appreciate your interest and support. To make the process as smooth as possible, please follow the guidelines below.
+
+## Requirements
+
+- You must have [Godot version 4.2.2](https://godotengine.org/download/archive/4.2.2-stable/) installed as this project is made in godot (Windows, Linux, Mac)
+- Git installed on you computer. Install git by [clicking here](https://git-scm.com/) (Windows, Linux, Mac)
+
+if you are a linux user you can also install Git through your package manager
+- Debian-based: 'sudo apt install git'
+- RedHat-based: 'sudo dnf install git'
+- Arch-based: 'sudo pacman -S git'
+- openSUSE: 'sudo zypper install git'
+- Alpine: 'sudo apk add git'
+- FreeBSD: 'sudo pkg install git'
+
+## How to contribute
+
+### Getting Started
+
+1. Fork the repository: Click the 'Fork' button at the top right corner of the repository page.
+
+2. Clone your forked repository:
+
+> git clone https://github.com/quellus/GDTuber.git
```

Comment:
For all of these block quotes, please change them to the code block format we have in the README

 ```sh
git clone https://github.com/quellus/GDTuber.git